### DISCUSSION TITLE
PROBLEMA ACCESO INGENIA

### DISCUSSION THREAD

**Discussion by Moorea-AI** [rramosp/ai4eng.v1#173] [00 - General]
Buenos dias profe Raúl,

Hoy nuevamente intento ingresar al portal de Ingenia y nuevamente tengo el acceso denegado. El problema ya lo habían "arreglado" desde la plataforma y estaba supuestamente "ok" pero hoy intento ingresar, tomando las precauciones previas al quiz para garantizar mi ingreso y no se me permite. Ya escribí pero también solicito su ayuda para que desde la universidad autoricen los accesos.

Muchas gracias,

**Comment by Felipe-RA** [comment]
Te mandamos el quiz como pdf al momento de comenzar

Lo deberás enviar  antes de las 10:30  a mi correo.

**Reply by Felipe-RA** [reply]
Necesitamos que nos des tu información de contacto de la UdeA @Moorea-AI
